aea/aea.py

```diff
@@ -157,6 +164,16 @@ def task_manager(self) -> TaskManager:
         """Get the task manager."""
         return self._task_manager
 
+    def _connect(self) -> None:
```

Comment:
This should be public not private imo. Also; it seems to me this should call a method in multiplexer like so 
```
connections = self.resources.get_all_connections()
self.multiplexer.setup(connections, default_routing)
```
as all this is scoped to the multiplexer

Reply:
done